static/js/containers/SettingsPage.js

```diff
@@ -0,0 +1,66 @@
+/* global SETTINGS: false */
+import React from 'react';
+import { connect } from 'react-redux';
+import Loader from 'react-loader';
+
+import Jumbotron from '../components/Jumbotron';
+import {
+  startProfileEdit,
+  FETCH_PROCESSING,
+  clearProfile,
+} from '../actions/index';
+import ProfileFormContainer from './ProfileFormContainer';
+
+class SettingsPage extends ProfileFormContainer {
+  componentDidMount() {
+    this.fetchProfile();
```

Comment:
@noisecapella  there is issue with that logic, when i dont use ``fetchProfile`` then in ``render()`` 
 ``const { profiles } = this.props;`` gets undefined. it seems app.js triggers after SettingsPage.js render